### PR TITLE
[Docs] Add timeout and fallback cheat sheets

### PR DESCRIPTION
**Pull request by peter-csala**
# Pull Request

## Details on the issue fix or feature implementation

Screenshot 2023-12-05 at 9 43 51

Screenshot 2023-11-22 at 20 52 49

Screenshot 2023-11-22 at 20 52 27

## Confirm the following

- [x]  I started this PR by branching from the head of the default branch
- [x]  I have targeted the PR to merge into the default branch
- [ ]  I have included unit tests for the issue/feature
- [x]  I have successfully run a local build

### CODE REVIEW

File: docs/community/cheat-sheets.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Cheat Sheets
+
+Cheat sheets provide easy to use code snippets for common scenarios.
+
+> [!NOTE]
+> The bellow cheat sheets are providing quick samples for the V8 API.
```
Comment (martincostello) [inline docs/community/cheat-sheets.md:6]:
```suggestion
> The below cheat sheets provide quick samples for the V8 API.
```

**Review by martincostello — COMMENTED** [review]
I haven't looked at the PDFs yet as I'm on mobile right now, but are you able to provide the mark-up used to create them in some way so we could regenerate them if needed in the future? Or are the PDFs themselves editable if you have the right software (which I don't, AFAIK).

**Comment by peter-csala** [comment]
> I haven't looked at the PDFs yet as I'm on mobile right now, but are you able to provide the mark-up used to create them in some way so we could regenerate them if needed in the future? Or are the PDFs themselves editable if you have the right software (which I don't, AFAIK).

I've generated the pdfs from powerpoint files. I know, I know ... there could be better ways like latex, but I'm not familiar with that.

**Comment by martincostello** [comment]
I've never used LaTeX either - I just see people in academia complain about it a lot 😆

File: docs/community/cheat-sheets.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+Cheat sheets provide easy to use code snippets for common scenarios.
+
+> [!NOTE]
+> The below cheat sheets provide quick samples for the V8 API.
+
+## Download
+
+| Strategy | Pdf link |
+| ------------- | :-------------: |
+| Timeout | [Download](../media/Timeout%20cheat%20sheet.pdf) |
```
Comment (martincostello) [inline docs/community/cheat-sheets.md:12]: Can you swap the spaces for dashes in the file names? Then we don't need to worry about encoding or quoting paths.

File: docs/media/Fallback-cheat-sheet.pdf
Comment (martincostello) [inline docs/media/Fallback-cheat-sheet.pdf:1]: Feedback on this file as bullet points as there's no PDF diff:

- `if the execution fail` => `if the execution fails`.
- `specify asynchronously delegate` => `specify asynchronous delegate`.
- Let's lowercase all the file names.

File: docs/media/Timeout-cheat-sheet.pdf
Comment (martincostello) [inline docs/media/Timeout-cheat-sheet.pdf:1]: `Specify asynchronously delegate for timeout notification` => `Specify asynchronous delegate for timeout notification`

File: docs/community/cheat-sheets.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+> [!NOTE]
+> The below cheat sheets provide quick samples for the V8 API.
+
+## Download
+
+| Strategy | Pdf link |
+| ------------- | :-------------: |
+| Timeout | [Download](../media/Timeout-cheat-sheet.pdf) |
+| Fallback | [Download](../media/Fallback-cheat-sheet.pdf) |
```
Comment (martincostello) [inline docs/community/cheat-sheets.md:13]:
```suggestion
| Timeout | [Download](../media/timeout-cheat-sheet.pdf) |
| Fallback | [Download](../media/fallback-cheat-sheet.pdf) |
```

**Review by martincostello — APPROVED** [review]
These are great! 👏

**Review by martintmk — APPROVED** [review]
These looks great! Thanks @peter-csala

**Comment by peter-csala** [comment]
@martincostello The lower casing did not worked really well :D 
- https://www.pollydocs.org/media/Timeout-cheat-sheet.pdf 👍🏻 
- https://www.pollydocs.org/media/timeout-cheat-sheet.pdf 👎🏻 

I'll create a fixing PR soon.

**Pull request merged**